examples/jsm/controls/MapControls.js

```diff
@@ -1,1166 +0,0 @@
-/**
- * @author qiao / https://github.com/qiao
```

Comment:
I don't think there's a reason to remove the already-transformed `jsm` files in this PR.

Reply:
The converted file will overwrite this, so I removed it.